tests/Maker/MakeEventTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Tests\Maker;
+
+use Symfony\Bundle\MakerBundle\Maker\MakeEvent;
+use Symfony\Bundle\MakerBundle\Test\MakerTestCase;
+use Symfony\Bundle\MakerBundle\Test\MakerTestRunner;
+
+class MakeEventTest extends MakerTestCase
+{
+    public function getTestDetails()
+    {
+        yield 'it_make_subscriber_without_conventional_name' => [$this->createMakerTest()
+            ->run(function (MakerTestRunner $runner) {
+                $output = $runner->runMaker(
+                    [
+                        'foo',
+                        // event class type
+                        'Subscriber',
+                        // event name
+                        'kernel.request'
+                    ]
+                );
+
+                self::assertFileExists($runner->getPath('src/EventSubscriber/FooSubscriber.php'));
+
+                $content = file_get_contents($runner->getPath('src/EventSubscriber/FooSubscriber.php'));
+
+                self::assertStringContainsString(
+                    'KernelEvents::REQUEST => \'onKernelRequest\'',
+                    $content
+                );
```

Comment:
Maybe we can compare generated code with a file in `Tests/Maker/fixtures/`, WDYT ?

Reply:
Updated if you want to check ;) [9c96bbf](https://github.com/symfony/maker-bundle/pull/1366/commits/9c96bbf350b7484cc8650e3f86bacde96c09ec6c)